pkg/graph/schema.resolvers.go

```diff
@@ -1590,6 +1590,31 @@ func (r *queryResolver) SystemIntakeContacts(ctx context.Context, id uuid.UUID)
 	if err != nil {
 		return nil, err
 	}
+
+	euaIDs := make([]string, len(contacts))
+	for i, contact := range contacts {
+		euaIDs[i] = contact.EUAUserID
+	}
+
+	userInfos, err := r.service.FetchUserInfos(ctx, euaIDs)
```

Comment:
Major bug here: this line throws a nil-dereference error, because `r.service.FetchUserInfos` doesn't get initialized in `pkg/server/routes.go`. I have fixed code on https://github.com/CMSgov/easi-app/tree/EASI-1971/FetchUserInfos-bugfix that you can merge if you want, or look at the [commit with the fix](https://github.com/CMSgov/easi-app/commit/692e03155ffe3ade021728b106af4525e891d145) to guide your own fix.